src/components/app/app.test.tsx

```diff
@@ -1,12 +1,18 @@
-import {shallow} from 'enzyme';
-import React from 'react';
+/**
+ * @jest-environment jsdom
+ */
 
-import {App} from './app';
+import { mount } from 'enzyme';
+import React from 'react';
+import { MemoryRouter } from 'react-router';
+import { MainUnauthorized } from '../main-unauthorized';
+import { App } from './app';
 
-describe(App, () => {
-  it('should parse component correctly', () => {
-    const markup = shallow(<App/>);
-    expect(markup.html()).toContain('app');
-    expect(markup.find('h1').html()).toContain('App');
-  });
+test('check if main page for unauthorized users is loading', () => {
```

Comment:
I think we should keep the nice features of `describe` and `it` jest functions. It is definitely useful for context, the output is more verbose and meaningful and we should try to be consistent, and not mix and match.

What do you think?